### ISSUE TITLE
Test nightly Che on OCP 4

### ISSUE THREAD

**Issue by Katka92** [eclipse-che/che#17701]
### Is your task related to a problem? Please describe.
We have tests running successfully against Che deployed on minishift and minikube. Once we run the same test suite against Hosted Che, we found out many different issues - test ones and product ones. So, it would be great if we could run nightly Che tests on OCP to catch some issue earlier.

### Describe the solution you'd like
There already are some jobs running on OCP 4.5:
https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/che-regular-tests/view/nightly/job/che-chectl-tls-oauth/
https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/che-regular-tests/view/nightly/job/che-chectl-tls-no-oauth/

It seems that some of them were not run for some time, so investigate if they're running correctly/fix them. 

### Describe alternatives you've considered
Maybe it would be worth to set that one as a nightly job instead of the one running against minikube.

### Additional context
Having nightly Che tested against OCP 4 could prevent us from finding issues when the version of a Hosted Che is updated.

**Comment by dmytro-ndp** [comment]
The issue is follow up of https://github.com/eclipse/che/issues/16752.

**Comment by ibuziuk** [comment]
From the Hosted Che perspective, it would be more suitable to run against OSD v4 (not OCP). Not sure if we have the infrastructure for this to use in the upstream though

**Comment by Katka92** [comment]
After some discussion, it seems that testing Che on OCP will not be that beneficial.
Testing on OSD v4 would require new infra and installing Che there can bring the problems (like we usually don't have rights to create CRDs etc). We think it is not worth it as Hosted Che should be replaced by Hosted Toolchain in few months.
Instead of that, we decided to run pre-release tests against compatibility PR in rh-che repo more often to catch the problems earlier. Closing the issue.
